frontend/src/metabase/query_builder/selectors.js

```diff
@@ -306,6 +306,12 @@ export const getOriginalQuestion = createSelector(
   (metadata, card) => metadata && card && new Question(card, metadata),
 );
 
+export const getOriginalQuestionWithParameterValues = createSelector(
+  [getMetadata, getOriginalCard, getParameterValues],
+  (metadata, card, parameterValues) =>
+    metadata && card && new Question(card, metadata, parameterValues),
+);
```

Comment:
There is a `getOriginalQuestion` selector, but it returns a question without any `parameterValues`. We need the `parameterValues` to tell whether the question really has "dirtied" or not when the query results are returned [here](https://github.com/metabase/metabase/pull/33211/files#diff-222944c63a643f354fcf9e8fd93575fd6429baa8d763755500ab25cfc50842e5R183).